ui/app/components/secret-create-or-update.js

```diff
@@ -93,12 +82,6 @@ export default class SecretCreateOrUpdate extends Component {
         ? set(this.validationMessages, name, `${name} can't be blank.`)
         : set(this.validationMessages, name, '');
     }
-    // check duplicate on path
-    if (name === 'path' && value) {
-      this.secretPaths?.includes(value)
-        ? set(this.validationMessages, name, `A secret with this ${name} already exists.`)
-        : set(this.validationMessages, name, '');
-    }
```

Comment:
Just want to double check that this is KV only? Seems like all secret types cannot have a path with the same name.